src/server/api/controllers/qr.controller.js

```diff
@@ -0,0 +1,22 @@
+const knex = require('../../config/db');
+
+const get = async (id) => {
+  try {
+    const instances = await knex('game_instances')
+      .select('game_instances.game_code as code')
```

Comment:
game_instances.game_code as code, code it's not understandable, maybe something like this would be more clear for others (game_instances.game_code as QRcode)

Reply:
Done